### DISCUSSION TITLE
channel_post_handler does not see posts made by another bot

### DISCUSSION THREAD

**Discussion by robertpustota** [eternnoir/pyTelegramBotAPI#2216] [General]
So I have this handler
```python
@bot.channel_post_handler(func=lambda message: True, content_types=["text", "document", "photo", "audio", "video", "voice"])
```
That works well and would catch a message if I sent it, but when it sends another bot, for some reason, this handler does not catch a message.

Can It be that Telegram has forbidden communication with bots, even in channels?

**Comment by Badiboy** [comment]
Have no additional information. May be Telegram improve something at backend...

**Reply by robertpustota** [reply]
There were 2 bots that worked well for a long time, but yesterday, without any actions from my side, my bot stopped catching messages from another bot in channel. 

I have to admit that when I test it in newly created channel, it works well again
